README.md

```diff
@@ -14,15 +14,61 @@ It is also possible to use this library to convert an SVG to a PNG before downlo
 ```javascript
 import downloadSvg from 'svg-crowbar';
 
-downloadSvg(document.querySelector('svg'), 'my_svg');
+downloadSvg(document.querySelector('svg'));
 ```    
 or
 ```javascript
 import { downloadPng } from 'svg-crowbar';
 
-downloadPng(document.querySelector('svg'));
+downloadPng(document.querySelector('svg'), 'my_svg', { css: 'internal' });
 ```
 
-Filename is determined by element id, class or page title, when not provided explicitly.
+The `downloadSVG`/`downloadPNG` functions each have three arguments:
 
-An error is thrown in case no valid SVG element was provided.
+```javascript
+downloadSVG([svgElement], [filename], [options])
```

Comment:
```suggestion
downloadSVG(svgElement, [filename], [options])
```
Square brackets feel like an indicator of an optional argument to me, I would omit them here and in in the next line